tools/lkl/include/lkl.h

```diff
@@ -229,6 +233,17 @@ struct lkl_netdev {
 int lkl_netdev_add(struct lkl_netdev *nd, void *mac);
 
 /**
+* lkl_netdevs_remove - destroy all network devices
```

Comment:
I think anyone that called lkl_netdev_add has to call lkl_netdevs_remove to avoid mem leak. Can you make it clear in the comment that one has to use "host_ops.host_clean = lkl_netdevs_remove"?